pilot/pkg/xds/eds.go

```diff
@@ -425,10 +427,16 @@ func (eds *EdsGenerator) Generate(proxy *model.Proxy, push *model.PushContext, w
 			eds.Server.Cache.Add(builder, req, resource)
 		}
 	}
-	return resources, model.XdsLogDetails{
+	return resources, removed, model.XdsLogDetails{
```

Comment:
The behavior here is pretty subtle. I think some comments here would help a lot, to explain why we have Delta for EDS/SDS despite them not actually being "delta"